tools/mo/openvino/tools/mo/main.py

```diff
@@ -313,21 +317,41 @@ def raise_ie_not_found():
     return argv
 
 
-def prepare_ir(argv):
-    argv = arguments_post_parsing(argv)
+def check_fallback(argv : argparse.Namespace):
+    fallback_reasons = {}
+    fallback_reasons['extensions'] = \
+        lambda argv : hasattr(argv, 'extensions') and argv.extensions is not None and len(argv.extensions) > 0
+    fallback_reasons['transformations_config'] = \
+        lambda argv: hasattr(argv, 'transformations_config') and argv.transformations_config is not None and len(argv.transformations_config) > 0
```

Comment:
Please, make sure that transformations_config path or extensions paths are not sent to analytics.

Reply:
The final messages contain only 'extensions' and 'transformations_config' string (the keys of `fallback_reasons` dict)